src/main/java/frc/robot/subsystems/swerve/Swerve.java

```diff
@@ -299,37 +327,32 @@ public boolean isAtPosition(Pose2d pose2d) {
     }
 
+    private boolean isAtTranslationPosition(
+            double currentTranslationPosition, double targetTranslationPosition, double currentTranslationVelocity
+    ) {
+        return Math.abs(currentTranslationPosition - targetTranslationPosition) < SwerveConstants.TRANSLATION_TOLERANCE_METERS && Math.abs(
+                currentTranslationVelocity) < SwerveConstants.TRANSLATION_VELOCITY_TOLERANCE;
+    }
+    
     public boolean isAtXAxisPosition(double xAxisPosition) {
-        return true;
-        //        final double currentXAxisVelocity = getFieldRelativeVelocity().vxMetersPerSecond;
-        //        return
-        // isAtTranslationPosition(RobotContainer.POSE_ESTIMATOR.getCurrentPose().toBlueAlliancePose().getX(),
-        // xAxisPosition, currentXAxisVelocity);
+        final double currentXAxisVelocity = getFieldRelativeVelocity().vxMetersPerSecond;
+        return isAtTranslationPosition(RobotContainer.POSE_ESTIMATOR.getCurrentPose().toBlueAlliancePose().getX(),
+                xAxisPosition,
+                currentXAxisVelocity
+        );
     }
 
     public boolean isAtYAxisPosition(double yAxisPosition) {
-        return true;
-        //        final double currentYAxisVelocity = getFieldRelativeVelocity().vyMetersPerSecond;
-        //        return
-        // isAtTranslationPosition(RobotContainer.POSE_ESTIMATOR.getCurrentPose().toBlueAlliancePose().getY(),
-        // yAxisPosition, currentYAxisVelocity);
+        final double currentYAxisVelocity = getFieldRelativeVelocity().vyMetersPerSecond;
+        return isAtTranslationPosition(RobotContainer.POSE_ESTIMATOR.getCurrentPose().toBlueAlliancePose().getY(),
+                yAxisPosition,
+                currentYAxisVelocity
+        );
     }
 
     public boolean isAtAngle(Rotation2d angle) {
-        return true;
-        //        return Math.abs(angle.getDegrees() -
-        // RobotContainer.POSE_ESTIMATOR.getCurrentPose().toBlueAlliancePose().getRotation().getDegrees()) <
-        // SwerveConstants.ROTATION_TOLERANCE.getDegrees()
-        //                &&
-        //                Math.abs(getSelfRelativeVelocity().omegaRadiansPerSecond) <
-        // SwerveConstants.ROTATION_VELOCITY_TOLERANCE;
-    }
-
-    private boolean isAtTranslationPosition(
-            double currentTranslationPosition, double targetTranslationPosition, double currentTranslationVelocity
-    ) {
-        return Math.abs(currentTranslationPosition - targetTranslationPosition) < SwerveConstants.TRANSLATION_TOLERANCE_METERS && Math.abs(
-                currentTranslationVelocity) < SwerveConstants.TRANSLATION_VELOCITY_TOLERANCE;
+        return Math.abs(angle.getDegrees() - RobotContainer.POSE_ESTIMATOR.getCurrentPose().toBlueAlliancePose().getRotation().getDegrees()) < SwerveConstants.ROTATION_TOLERANCE.getDegrees() && Math.abs(
+                getSelfRelativeVelocity().omegaRadiansPerSecond) < SwerveConstants.ROTATION_VELOCITY_TOLERANCE;
```

Comment:
this is unreadable, consider saving the booleans by themselves and returning the `&&` of them